weave

```diff
@@ -1552,6 +1571,9 @@ launch_proxy() {
         --entrypoint=/home/weave/weaveproxy \
         $WEAVEPROXY_DOCKER_ARGS $EXEC_IMAGE $COVERAGE_ARGS $PROXY_ARGS)
     wait_for_status $PROXY_CONTAINER_NAME http_call_unix $PROXY_CONTAINER_NAME status.sock
+    for sock in $PROXY_VAR_RUN_SYMLINKS; do
+      ln -sf -T "/var/run/weave/$sock" "/host/var/run/$sock"
```

Comment:
It's to place a symlink at `/var/run/$sock` pointing to `/var/run/weave/$sock`. and `-T` means "Target".